enterprise/backend/src/metabase_enterprise/audit_db.clj

```diff
@@ -135,8 +135,8 @@
         (u.files/unzip-file analytics-zip-resource
                             (fn [entry-name]
                               (str/replace entry-name
-                                           "resources/instance_analytics/"
-                                           "plugins/instance_analytics/")))
+                                           "instance_analytics/"
+                                           (str "plugins" "/instance_analytics/"))))
```

Comment:
not sure I understand why this is wrapped in `str`? did you mean to change it to use `plugin-dir`?